pkg/kubecfg/vm.go

```diff
@@ -231,22 +232,45 @@ func dirURL(path string) *url.URL {
 	return &url.URL{Scheme: "file", Path: path}
 }
 
+func buildOverlayObject(format, flag string) string {
+	re := regexp.MustCompile(`^([^$=]*)=(.*)$`)
+	parts := re.FindStringSubmatch(flag)
```

Comment:
`FindStringSubmatch` can also return `nil`, I'm not sure whether we actually need to handle that case though considering the passed `flag` automatically fails if you give it nothing.

Raising awareness to it nonetheless :smile:

Reply:
`len(slice)` when the slice is nil yields 0